src/applications/caregivers/containers/IntroductionPage.jsx

```diff
@@ -2,9 +2,7 @@ import React, { useCallback, useEffect } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
-import Telephone, {
-  CONTACTS,
-} from '@department-of-veterans-affairs/component-library/Telephone';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/Telephone';
```

Comment:
This will change in the future - see https://github.com/department-of-veterans-affairs/component-library/pull/295/files#diff-fe2bb4919a004e498f2910f59dea2f6e47b461cf54f89a3be48ccd0f7cb86d4b (not merged yet). You should be okay until they remove the React `Telephone` component